README.md

```diff
@@ -32,19 +40,19 @@ sudo make
 To add more workers:
 
 ```bash
-docker-compose up -d --scale worker=5 --no-recreate
+docker compose up -d --scale worker=5 --no-recreate
```

Comment:
I thought the hyphen was the right way -- that being deprecated?

Reply:
That's correct, it has been deprecated. See https://docs.docker.com/compose/compose-v2/